mantidproject/mantidimaging
# Duplicate context menu in the a new main window menu

Fixes #676 

2 Things have happened in this PR:
- I have disabled most main window options until a stack is loaded, if no stacks are present they should be disabled, and re-enabled later on.
- The context menu for a loaded stack, should also be available in a menu called Image at the top of the Main window

To Test:
- Open imaging, ensure that Image, Workflow, File->Save, File->LoadSampleLog, File->Load180DegreeProjection are disabled.
- Load a stack
- Ensure that all of the disabled items are now enabled.
- Check that the Image menu only applies operations/actions to the currently viewed stack.
